linodecli/operation.py

```diff
@@ -6,12 +6,20 @@
 import glob
 import json
 import platform
+from dataclasses import dataclass
 from getpass import getpass
 from os import environ, path
 
 from linodecli.helpers import handle_url_overrides
 
 
+@dataclass
```

Comment:
`dataclass` decorator seems unnecessary here?

Reply:
I added `@dataclass` because the linter was complaining about too few public methods with a normal class. It might just be better for me to add a linter ignore here 👍

Reply:
Haha, I see